pkg/http/http.go

```diff
@@ -29,26 +35,97 @@ type ClientOptions struct {
 // Sender provides an interface to the piper http client for uid/pwd authenticated requests
 type Sender interface {
 	SendRequest(method, url string, body io.Reader, header http.Header, cookies []*http.Cookie) (*http.Response, error)
+	UploadFile(url, file, fieldName string, header http.Header, cookies []*http.Cookie) (*http.Response, error)
```

Comment:
Maybe introduce a new interface instead: `Uploader`?

Then we do not need to implement this function in all http usages ...